packages/hig-toast/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@hig/toast",
+  "version": "0.1.0",
+  "description": "Floating message boxes that report a system occurrence",
+  "devDependencies": {
+    "@hig/styles": "^0.1.0-alpha"
+  },
+  "main": "src/index.js",
```

Comment:
For extra credit, Rollup and Webpack also look to a "module" field in package.json for an ESM version of the export. We could set this as well.

https://github.com/rollup/rollup/wiki/pkg.module